tests/base.py

```diff
@@ -177,8 +177,11 @@ def setupDirs(self, test_name):
         """
         Set up directories for testing.
         """
-        # FIXME: Item "None" of "Optional[Any]" has no attribute "lower"mypy(error)
-        if self.hpss_path.lower() == "none":  # type: ignore
+        if self.hpss_path:
+            hpss_path = self.hpss_path
+        else:
+            raise Exception("Invalid self.hpss_path={}".format(self.hpss_path))
```

Comment:
I think `mypy` is smart enough to determine that after this exception, `self.hpss_path` can't be `None`, but I think using a new variable not of type `Optional` in the alternative case improves readability.